core/src/test/scala/unit/kafka/log/LogConfigTest.scala

```diff
@@ -402,6 +403,22 @@ class LogConfigTest {
     }
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("retain", "delete"))
+  def testValidRemoteLogRetentionPolicy(policy: String): Unit = {
```

Comment:
please update the test name

(eg) testValidRemoteLogDisablePolicy (or) something on similar lines